gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1312,6 +1312,44 @@ public void xattr_statistics() throws IOException {
     assertThat(myGhfs.delete(testRoot, true)).isTrue();
   }
 
+  public void http_IOstatistics() throws IOException {
+    FSDataOutputStream fout = ghfs.create(new Path("/file1"));
+    fout.writeBytes("Test Content");
+    fout.close();
+
+    assertThat(
+            ((GoogleHadoopFileSystem) ghfs)
+                .getIOStatistics()
+                .counters()
+                .get(INVOCATION_CREATE.getSymbol()))
+        .isEqualTo(1);
+    assertThat(
+            ((GoogleHadoopFileSystem) ghfs)
+                .getIOStatistics()
+                .counters()
+                .get(GhfsStatistic.ACTION_HTTP_GET_REQUEST.getSymbol()))
+        .isEqualTo(2);
+    assertThat(
+            ((GoogleHadoopFileSystem) ghfs)
+                .getIOStatistics()
+                .counters()
+                .get(GhfsStatistic.ACTION_HTTP_PUT_REQUEST.getSymbol()))
+        .isEqualTo(1);
+    assertThat(
+            ((GoogleHadoopFileSystem) ghfs)
+                .getIOStatistics()
+                .counters()
+                .get(GhfsStatistic.ACTION_HTTP_PATCH_REQUEST.getSymbol()))
+        .isEqualTo(1);
+    assertThat(ghfs.delete(new Path("/file1"))).isTrue();
+    assertThat(
+            ((GoogleHadoopFileSystem) ghfs)
+                .getIOStatistics()
+                .counters()
+                .get(GhfsStatistic.ACTION_HTTP_DELETE_REQUEST.getSymbol()))
+        .isEqualTo(1);
```

Comment:
Do we have a delete call when a file is created ?

Reply:
Please add comments for what the requests for 
ex : PUT request could be create object api ?

Reply:
Added the comments